README.md

```diff
@@ -1,6 +1,7 @@
 # Prism
 
-<a href="https://codeclimate.com/github/stoplightio/prism/test_coverage"><img src="https://api.codeclimate.com/v1/badges/f5e363a7eb5b8f4e570f/test_coverage" /></a>
+[![Build Status](https://dev.azure.com/vncz/vncz/_apis/build/status/stoplightio.prism?branchName=master)](https://dev.azure.com/vncz/vncz/_build/latest?definitionId=1&branchName=master)
```

Comment:
Can you make a Jira task for moving this off your personal account?

Reply:
Yeah good point.